app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```

đọc lại coding convention 
https://github.com/framgia/coding-standards/blob/master/vn/rails/standard.md